Mathlib/CategoryTheory/Equivalence.lean

```diff
@@ -110,6 +110,79 @@ abbrev unitInv (e : C ≌ D) : e.functor ⋙ e.inverse ⟶ 𝟭 C :=
 abbrev counitInv (e : C ≌ D) : 𝟭 D ⟶ e.inverse ⋙ e.functor :=
   e.counitIso.inv
 
+section CategoryStructure
+
+/-- A morphism between equivalences of categories is a natural transformation between their
+functors. -/
+def Hom : (C ≌ D) → (C ≌ D) → Type (max u₁ v₂) :=
+  fun f g ↦ (f.functor ⟶ g.functor)
```

Comment:
As this is not a structure, I do not think it needs to be a separate definition, `f.functor ⟶ g.functor` could be used directly in the definition of the `Category` instance.